src/github/nag.rs

```diff
@@ -339,6 +369,17 @@ fn evaluate_nags() -> DashResult<()> {
     };
 
     for mut proposal in finished_fcps {
+        let initiator = match githubuser::table
+                  .find(proposal.fk_initiator)
+                  .first::<GitHubUser>(conn) {
+            Ok(i) => i,
+            Err(why) => {
+                error!("Unable to retrieve proposal initiator for proposal id {}: {:?}",
```

Comment:
I need to double check the schema, but I'm pretty sure the database ensures this never happens and so this can probably be safely written as an expect.

Reply:
For now, to be on the safe side I'll keep the match. I'll change later if you are sure. :)